src/merge.py

```
@@ -423,24 +456,36 @@ def main():
     icloud_events = _collect_icloud_events(all_icloud_events, skip_days)
 
     now = datetime.now()
-    today_bod = datetime(now.year, now.month , now.day, 0, 0, 0)
+    today_bod = datetime(now.year, now.month, now.day, 0, 0, 0)
```

Ruff (0.15.10)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)